starters/e2e/qwikcity/locale.e2e.ts

```diff
@@ -1,12 +1,11 @@
 import { expect, test } from "@playwright/test";
 
 test.describe("Qwik City locale API", () => {
-  // TODO(v2): fix this
-  test.skip("pass locale to Qwik", async ({ page }) => {
+  test("pass locale to Qwik", async ({ page }) => {
     await page.goto("/qwikcity-test/locale");
     const locale = page.locator(".locale");
-    const qContainer = page.locator("[q\\:container]");
     await expect(locale).toHaveText("test-locale");
+    const qContainer = page.locator("[q\\:container=paused]");
```

Comment:
yeah this is a good way to solve this, but I think we should use the same selector as the rest of the system
Look here https://github.com/QwikDev/qwik/blob/3feb2e6da2e4e9725dd8331e2730ba13e88e63d5/packages/qwik/src/core/util/markers.ts#L42

Reply:
good idea!

Reply:
done